frontend/src/shared/components/mapVoteButton/MapVoteButton.tsx

```
@@ -0,0 +1,28 @@
+
+function MapVoteButton({ onClick }: MapVoteButtonProps) {
+  return (
+    <button
+      css={[
+        inline_flex({ justify: 'center', align: 'center', gap: 4 }),
+        shadow.map,
+        mapVoteButton.base(),
+        mapVoteButton.floating(),
+      ]}
+      onClick={onClick}
+    >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)